executioner/runtimes/languages/gcc/entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Compile the source code
+ERROR=$(g++ -O2 -static src/main.cpp -o ./main 2>&1)
```

Comment:
We also need to be aware of whether we are compiling C++23 or C++17 code, for example, probably the same for other languages.

Reply:
Not important ATM